packages/manager/apps/pci-instances/src/components/tab/TabsPanel.component.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC, useMemo } from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import { Badge } from '@datatr-ux/uxlib';
+import { clsx } from 'clsx';
+
+type TabItem = {
+  label: string;
+  to: string;
+  count?: number;
+};
+
+const TabsPanel: FC<{ tabs: TabItem[] }> = ({ tabs }) => {
+  const location = useLocation();
+
+  const activeTab = useMemo(
+    () => tabs.find((tab) => location.pathname === tab.to),
+    [location.pathname, tabs],
+  );
+
+  return (
+    <div className="flex border-b border-[#e6f5fc]">
+      {tabs.map((tab, index) => (
+        <NavLink
+          key={`${tab.label}-${index}`}
```

Comment:
Either label is unique and you should use it or else you should only have the index but both don't make sense for a key